ansible/cluster-setup/site.yaml

```diff
@@ -340,4 +340,13 @@
           enabled: true
           url: "https://api{{ cluster_host }}:6443"
           token: "{{ svc_account_token }}"
+          tektonEnabled: true
+        githubOrgProvider:
+          enabled: true
+        ocm:
+          enabled: false
+        sonarqube:
+          enabled: false
+        techdocs:
+          enabled: false
```

Comment:
Why is techdocs disabled?

Reply:
As far as I can tell tech docs have not been working correctly with our demo up to this point, my thought is that we should just disable it for now, and when someone takes on the task to fix the techdocs in our demo/template, they'll enable them at that time along with the fix

Reply:
Sounds good